docs/articles/commands/text/custom_prefix_handler.md

```diff
@@ -0,0 +1,97 @@
+---
+uid: articles.commands.custom_prefix_handler
+title: Custom Prefix Handler
+---
+
+# Adding dynamic prefixes
+
+Prefixes are commonplace among Discord bots, and are used to determine if a message is a command or not. By default, DSharpPlus uses the `!` prefix, but you can change this to whatever you want. However, what if you want to have different prefixes for different servers? This is where custom prefix handlers come in.
+
+There are two manners of going about this; you can either use a prefix resolver delegate, or implement the `IPrefixResolver` interface. The former is simpler, but the latter is more powerful.
```

Comment:
maybe mention here why it is more powerful, particularly if you use tabs here - fwiw, i think this article works fine using subheadings instead of tags